src/components/IssueQueueItem.vue

```diff
@@ -10,6 +10,7 @@ const props = defineProps<{
 const isEmbargoed = computed(() => DateTime.fromISO(props.issue.unembargo_dt).diffNow().milliseconds > 0);
 const hasBadges = computed(() => isEmbargoed.value);
 const formattedDate = computed(() => DateTime.fromISO(props.issue.created_dt).toFormat('yyyy-MM-dd'));
+const pathID = computed(() => props.issue.cve_id || props.issue.uuid);
```

Comment:
```suggestion
const cveOrUuid = computed(() => props.issue.cve_id || props.issue.uuid);
```

Reply:
Thanks! I am doing a rebase as well so I am going to include this change as well to keep git history more clean.